openrouteservice-api-tests/conf/app.config.test

```diff
@@ -191,6 +201,13 @@
                           threads: 1,
                           weightings: "recommended,shortest",
                           landmarks: 32
+                        },
+                      fastisochrone:
```

Comment:
I am aware that this placement is counter-intuitive. The reason for it is that that `services/isochrones` is treated separately from the `routing` part in our code. The `isochrones` are not read for preprocessing, they only give some static constants. The information under routing is used for preparation. I am not sure if this can be easily changed or adapted. It is a general problem of how the app.config is read and the information processed. Somehow, it is neither done nor available in one spot.

Reply:
It seems entirely feasible to me though. The routing parameters used for preparation are set in `RoutingManagerConfiguration` through a call to `RoutingServiceSettings`. Isn't it possible to similarly query `IsochroneServiceSettings` there?

Reply:
In principle it is. But this introduces a whole new set of problems. How do we deal with the case that fast isochrones is enabled for a profile, but otherwise its disabled? ORS does not have a system for this in place. Either a profile is enabled or it is not. There is no enabled for this, but not for that. I will see what I can do.

Reply:
I see what you mean. Thanks for looking into this!

Reply:
My suggestion is to have something like this 
```profiles: {
            default_params: {
              enabled: false,
              threads: 12,
              weightings: "recommended",
              maxcellnodes: 5000
            },
            profile-vehicles-hgv: {
              enabled: true,
              threads: 12,
              weightings: "recommended",
              maxcellnodes: 3000
            }
          }
        },
```
in the `isochrones.fastisochrones` part of the config. *But* they will only be enabled if they are markes as active in `routing.profiles.active`. This is a compromise without creating a whole new system of enabled profiles. Would that work for you?

Reply:
Absolutely, this is exactly what I have envisioned. Cheers!

Reply:
Okay, done